lib/pbench/test/unit/server/test_utils.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from pbench.server.utils import filesize_bytes
+
+
+_sizes = [("  10  ", 10)]
+for i, mult in [
+    ("B", 1),
```

Comment:
Since you added support for an integer without units, you should add a test case for that!

Reply:
I thought I had done that at line 5?